### ISSUE TITLE
Path parameters

### ISSUE THREAD

**Issue by mzawitowski** [SamWSoftware/serverless-auto-swagger#60]
Plugin version 2.4.0.

I was able to present path params only by duplicating the path:
```
handler: src/handler.hello
    events:
      - httpApi:
          path: hello/{helloId}
          method: post
          parameters:
            path: hello/{helloId}
```

Btw. no idea how to add any description to them.

**Comment by ronie-san** [comment]
The `pathToParam` function inside `ServerlessAutoSwagger` doesn't have the `description` param and neither that function has the implementation.

But theoretically, you could this:
```yml
handler: src/handler.hello
    events:
      - httpApi:
          path: hello/{helloId}
          method: post
          parameters:
            path:
              helloId:
                type: string
                description: Hello ID
```

**Comment by bfaulk96** [comment]
The expected behavior is that path parameters are generated from the http or httpApi path as far as I understand:

```yaml
      - httpApi:
          path: hello/{helloId}
```

But as you mentioned, that does now seem to be broken for 2.4.0. I'll look into it.

**Comment by bfaulk96** [comment]
This was actually caused by https://github.com/SamWSoftware/serverless-auto-swagger/pull/53, I found the issue I believe. Testing locally.

**Comment by bfaulk96** [comment]
Fixed in `2.4.1`

**Comment by mzawitowski** [comment]
A good idea would be a simple e2e test that will compare an already generated swagger.json file (snapshot) with the result of plugin work. One test and in most of the cases we can avoid regression.
